tests/Unit/NumericalAlgorithms/Interpolation/Python/Test_RegularGridInterpolant.py

```diff
@@ -0,0 +1,89 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import unittest
+from spectre import Spectral
+from spectre import DataStructures
+from spectre import Interpolation
+from numpy.polynomial.legendre import Legendre
+import numpy as np
+
+
+class TestRegularGrid(unittest.TestCase):
+
+    # random polynomial functions of low order for exact interpolation
```

Comment:
I suggest "random" -> "arbitrary", because it's not random in the sense of being randomly generated each time.